Live-Snap/Live-Snap/Networking/SnapcodeEndpointRequest.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SnapcodeEndpointRequest.swift
+//  Live-Snap
+//
+//  Created by Oleg Abalonski on 1/6/18.
+//  Copyright © 2018 Oleg Abalonski. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class SnapcodeEndpointRequest {
+    
+    var snapchatUsername: String
+    
+    init(snapchatUsername: String) {
+        self.snapchatUsername = snapchatUsername
+    }
+    
+    func start(completion: @escaping ((UIImage?) -> ())) {
+        
+        let url = "https://feelinsonice-hrd.appspot.com/web/deeplink/snapcode?username=\(snapchatUsername)&type=PNG&size=800"
+    
+        Alamofire.request(url).responseData { (response: DataResponse<Data>) in
+
+            response.result.ifSuccess {
+                if let imageData = response.result.value {
+                    let image = UIImage(data: imageData)
+                    completion(image)
+            }
+            }
+            
+            response.result.ifFailure {
```

Comment:
should print out more information about the request such as the url we tried to reach and failed, the status code received from the server, and the actual error of why the request failed